src/engraving/libmscore/staff.h

```diff
@@ -59,8 +59,10 @@ enum class Key;
 //---------------------------------------------------------
 
 struct SwingParameters {
-    int swingUnit;
-    int swingRatio;
+    int swingUnit = 0;
+    int swingRatio = 0;
+
+    bool isValid() const { return swingRatio >= 50 && swingUnit != 0; }
```

Comment:
The `>= 50` should not be necessary, since some people desire to have "reversed" swing, see for example https://musescore.org/en/node/333867. Note that the "TripletFeel" element (added as part of the GP import task) has such a "reversed" swing option, so we should also support the playback of it. 

(We might need to tweak the formula in `Swing::swingAdjustParams`, but it looks like we don't need to.)

Reply:
fixed